src/server/scripts/Zandalar/Underrot/underrot.h

```diff
@@ -0,0 +1,60 @@
+/*
+ * This file is part of the TrinityCore Project. See AUTHORS file for Copyright information
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation; either version 2 of the License, or (at your
+ * option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef DEF_UNDERROT_H_
+#define DEF_UNDERROT_H_
+
+#include "CreatureAIImpl.h"
+
+#define DataHeader "Underrot"
```

Comment:
This is not how we name our DataHeaders for instances. We normally use abbreviations of the original name such as "SOD" for Sanctum of Domination. If I recall correctly, UR was the name used by players, so it can be reused.

Reply:
we will reach collisions if we only use shortened names (e.g. Tomb of Sargeras, Temple of Sethraliss); using full name is just fine here